fern/definition/medical/fhir.yml

```diff
@@ -63,6 +75,32 @@ service:
           response:
             body: $GetConsolidatedQueryStatusResponse.Example0
 
+    createPatientConsolidated:
+      display-name: Create Patient's Consolidated Data
+      docs: Create Patient's Consolidated Data with a FHIR Bundle.
+      method: PUT
+      path: /patient/{id}/consolidated
+      path-parameters:
+        id:
+          type: string
+          docs: The ID of the Patient
+      request:
+        body: ConsolidatedBundleUpload
+      response:
+        type: fhir.Bundle
+      # examples:
+      #   - name: Example1
```

Comment:
waiting on deep to get back to me on the fix here since this example doesnt compile